src/tests/ilasm/MethodImplOptions/MethodImplOptionsTests.cs

```diff
@@ -17,12 +21,12 @@ public static int Main()
         if (string.IsNullOrWhiteSpace(coreRoot))
         {
             Console.WriteLine("Environment variable is not set: 'CORE_ROOT'");
```

Comment:
Do we need to keep the `Console.WriteLine()` statements here if we're already throwing an exception with the same message?

Reply:
The Console.WriteLine statements can sometimes show up better in the console logs. I wanted to limit the amount of changes I made here to avoid breaking workflows if possible.

Reply:
Ok that makes sense. Thanks for clarifying!